### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by ShuzhaoXie** [nerfstudio-project/nerfstudio#2701]
**Describe the bug**
```pythonTraceback (most recent call last):
  File "/home/szxie/anaconda3/envs/nerfstudio/bin/ns-train", line 8, in <module>
    sys.exit(entrypoint())
  File "/home/szxie/nerfstudio/nerfstudio/scripts/train.py", line 262, in entrypoint
    main(
  File "/home/szxie/nerfstudio/nerfstudio/scripts/train.py", line 247, in main
    launch(
  File "/home/szxie/nerfstudio/nerfstudio/scripts/train.py", line 189, in launch
    main_func(local_rank=0, world_size=world_size, config=config)
  File "/home/szxie/nerfstudio/nerfstudio/scripts/train.py", line 99, in train_loop
    trainer.setup()
  File "/home/szxie/nerfstudio/nerfstudio/engine/trainer.py", line 176, in setup
    self.viewer_state = ViewerBetaState(
  File "/home/szxie/nerfstudio/nerfstudio/viewer_beta/viewer.py", line 228, in __init__
    self.viewer_elements.extend(parse_object(pipeline, ViewerElement, "Custom Elements"))
  File "/home/szxie/nerfstudio/nerfstudio/viewer_beta/utils.py", line 143, in parse_object
    lower_rets = parse_object(v, type_check, new_tree_stub)
  File "/home/szxie/nerfstudio/nerfstudio/viewer_beta/utils.py", line 131, in parse_object
    obj_props = [(k, getattr(obj, k)) for k in dir(obj)]
  File "/home/szxie/nerfstudio/nerfstudio/viewer_beta/utils.py", line 131, in <listcomp>
    obj_props = [(k, getattr(obj, k)) for k in dir(obj)]
  File "/home/szxie/nerfstudio/nerfstudio/data/datamanagers/full_images_datamanager.py", line 347, in fixed_indices_eval_dataloader
    data[i]["image"] = data[i]["image"].to(self.device)
IndexError: list index out of range
```
**To Reproduce**
run:
```python
ns-train gaussian-splatting blender-data --data nerf_data/nerf_synthetic/lego
```

**Comment by KevinXu02** [comment]
I've fixed it in #2704.

**Comment by ShuzhaoXie** [comment]
> I've fixed it in #2704.

Thanks!
